src/main/java/com/redhat/cloud/common/clowder/configsource/handlers/QuarkusDataSourceClowderPropertyHandler.java

```diff
@@ -59,10 +71,16 @@ public String handle(String property, ClowderConfigSource configSource) {
 
             jdbcUrl = String.format("jdbc:%s%s", tracing, hostPortDb);
             if (useSsl) {
-                jdbcUrl = jdbcUrl + "?sslmode=" + sslMode;
+                params.put("sslmode", sslMode);
             }
             if (verifyFull) {
-                jdbcUrl = jdbcUrl + "&sslrootcert=" + createTempRdsCertFile(clowderConfig.database.rdsCa);
+                params.put("sslrootcert", createTempRdsCertFile(clowderConfig.database.rdsCa));
+            }
+            if (!params.isEmpty()) {
+                StringBuilder builder = new StringBuilder();
+                params.keySet().forEach(x -> builder.append(String.format("%s=%s&", x, params.get(x))));
+                builder.setLength(builder.length() - 1);
+                jdbcUrl = String.format("%s?%s", jdbcUrl, builder);
```

Comment:
The `+` operator will concatenate the strings at a lower cost than `String.format`. Streams could also be used to avoid the `builder.setLength(builder.length() - 1);` move which may not be obvious to someone unfamiliar with the code. None of that is mandatory from my perspective though, so feel free to pick whichever approach you prefer :smile:
```suggestion
                String paramsString = params.entrySet().stream()
                    .map(entry -> entry.getKey() + "=" + entry.getValue())
                    .collect(Collectors.joining("&"));
                jdbcUrl += "?" + paramsString;
```

Reply:
I'll keep what I have.